Content/workflows/liquibase-community/upgrading-liquibase.htm

```diff
@@ -6,64 +6,42 @@
-            <li>	If you have installed Java, however, you are still receiving an error, add Java to your PATHs using the steps provided in the <a href="https://docs.liquibase.com/concepts/installation/home.html">Verifying</a> section for your operating system.</li>
-        </ul>
-        <h3>Verification #4: Run the <MadCap:variable name="General.Liquibase" /> <code>version</code> command</h3>
-        <p>To verify that you upgraded <MadCap:variable name="General.Liquibase" /> to the expected version, run the <MadCap:variable name="General.Liquibase" /> <code>version</code> command.</p>
-        <ol>
-            <li>Open your Linux Terminal or Windows Command Prompt.</li>
-            <li>Run the following:</li><pre xml:space="preserve"><code class="language-text">liquibase --version</code></pre>
-        </ol>
-        <h3>Troubleshooting</h3>
+        <p>Before upgrading <MadCap:variable name="General.Liquibase" />, read the latest <MadCap:xref href="../../release-notes/home.htm">Release Notes</MadCap:xref> for any changes that may affect your existing setup.</p>
+
+        <p>Upgrading <MadCap:variable name="General.Liquibase" /> follows the <a href="../../install/home.htm">Installing [%=General.Liquibase%]</a> instructions, but in one of two ways:</p>
         <ul>
-            <li>If you still see the old version of <MadCap:variable name="General.Liquibase" />, ensure that the new <MadCap:variable name="General.Liquibase" /> folder you downloaded is set to your PATHs.</li>
-            <li>If the PATH is pointing to the new version, then you may need to restart your command line to load the new PATH.</li>
+            <li>Overwrite the existing <MadCap:variable name="General.Liquibase" /> installation directory</li>
+            <li>Install to a new installation directory</li>
         </ul>
-        <h3>Verification #5: Run the <MadCap:variable name="General.Liquibase" /> <code>validate</code> command</h3>
-        <p>To verify that your existing project <MadCap:variable name="General.changelog" />s are compatible with your current projects, you can run the <MadCap:variable name="General.Liquibase" /> <code>validate</code> command.</p>
+
+        <h3>New Installation Directory</h3>
+        <p>Installing to a new directory allows to have multiple versions of Liquibase you can easily switch between. But may require additional configuration changes.</p>
+
+        <p>If you install to a new directory, make sure you:</p>
         <ol>
-            <li>Open your Linux Terminal or Windows Command Prompt.</li>
-            <li>Run the following:</li><pre xml:space="preserve"><code class="language-text">liquibase validate</code></pre>
+            <li>Copy any files from the <code>lib</code> directory from the old directory to the new.</li>
```

Comment:
Any files? Don't we want to advise not to copy older drivers unless they need them? I understand (I think?) that we're going for unversioned filenames for drivers in the future but a lot of people have Liquibase installed already and may not realize they do not need to copy all of the drivers from the old lib to the new lib.

Reply:
With 4.11+ the lib directory will only be files they explicitly wanted, so I tried to keep this the simple case. People coming from older versions may have old version they don't want to copy over and I added a note in there somewhere about that case. But that note is a bit separate in order to keep this "standard"/simple.